dracut/driver_updates.py

```diff
@@ -625,10 +625,17 @@ def main(args):
 
     if mode in ('--disk', '--net'):
         request, dev = args
+
+        # Guess whether this is an ISO or RPM based on the filename.
+        # If neither matches, assume it is a device node and processes as an ISO.
+        # This is relevant for both --disk and --net since --disk could be
+        # pointing to files within the initramfs.
         if dev.endswith(".iso"):
             process_driver_disk(dev)
-        else:
+        elif dev.endswith(".rpm"):
```

Comment:
I'm not sure I'm getting the commit 14c4fa right, but wasn't the purpose of the change to allow for installation of multiple DD RPMs from a given (NFS) directory?